conmon-rs/server/src/config.rs

```diff
@@ -125,6 +125,16 @@ pub struct Config {
     )]
     /// OpenTelemetry GRPC endpoint to be used for tracing.
     tracing_endpoint: String,
+
+    #[get_copy = "pub"]
+    #[arg(
+        default_value_t,
+        env(concat!(prefix!(), "SHUTDOWN_DELAY")),
+        long("shutdown-delay"),
+        value_name("DELAY"),
+    )]
+    /// Automatically stop conmon-rs after DELAY seconds of inactivity. (0 = disabled)
+    shutdown_delay: f64,
```

Comment:
I think we should make this something like [Option<`humantime::Duration`>](https://docs.rs/humantime/latest/humantime/struct.Duration.html)

Reply:
Hmm... the string representation of a go `time.Duration` is not compatible with the `humantime::Duration` parser. But we could write a custom to string method in go. Do we expect the conmon server to be invoked by humans or only by the go client?

Reply:
IMO we should only support go client

Reply:
And even for humans it is still relatively ergonomic to specify the duration in (fractional) seconds.  
`1.5` vs `1s 500ms`  
Especially considering that I expect the most common durations to be in the range of a few full seconds.

Reply:
Is specifying milliseconds really a use case? I'm also fine with using seconds only.